web_fragments/templates/web_fragments/standalone_fragment.html

```diff
@@ -0,0 +1,9 @@
+<html>
```

Comment:
Good catch. That was a refactoring/rebase gone bad. I've deleted standalone_component.html which was the old file name.